pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -77,6 +77,93 @@ The following services and service components have been upgraded to the listed v
 - traefik-forward-auth: 0.3.2
 - velero: 3.1.3
 
+## Known Issues 
+
+The following items are known issues with this release.
+
+### Create cert-manager resources on attached clusters with cert-manager pre-installed
+
+If you attach a cluster that already has `cert-manager` installed, you need to manually create the three cert-manager resources, in the yaml file, after attaching your cluster.
+
+For example, [Konvoy-created clusters that are self-managed][konvoy-self-managed] have `cert-manager` already installed to the `cert-manager` namespace.
+
+Verify that your cluster is self-managed by:
+
+```bash
+export KUBECONFIG=<kubeconfig-path>
+export WORKSPACE_NAMESPACE=<your-workspace-namespace>
+kubectl get kommandercluster -n $WORKSPACE_NAMESPACE
+```
+
+If your cluster is not self-managed, the output resembles this example:
+
+```bash
+error: the server doesn't have a resource type "kommandercluster"
+```
+
+If your cluster is self-managed, the output resembles this example:
+
+```bash
+NAME     AGE   PHASE
+my-cluster   30m   Provisioned
+```
+
+If your cluster is self-managed, then create the following yaml file:
```

Comment:
what should happen if the cluster is not self-managed?

Reply:
@Colin1841 changed it to this way and original one was "Create the following yaml file only if your cluster is self-managed:"

If not self-managed, you don't need to apply this yaml